### PR TITLE
SW-4382 List Inventory by Species

### PR DESCRIPTION
**Pull request by charlie-tf**
- List inventory by species, which is the same as the V1 list
- Mostly copy/pasted from V1
- Placeholders for by-nursery and by-batch

image
image
image

### CODE REVIEW

File: src/components/InventoryV2/FilterUtils.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```
Comment (charlie-tf) [inline src/components/InventoryV2/FilterUtils.ts:1]: copied from V1

File: src/components/InventoryV2/InventoryCellRenderer.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import React from 'react';
```
Comment (charlie-tf) [inline src/components/InventoryV2/InventoryCellRenderer.tsx:1]: copied from V1

File: src/components/InventoryV2/InventoryFiltersPopover.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
+import { Popover, Theme, Typography } from '@mui/material';
```
Comment (charlie-tf) [inline src/components/InventoryV2/InventoryFiltersPopover.tsx:1]: copied from V1

File: src/components/InventoryV2/InventoryTable.tsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
```
Comment (charlie-tf) [inline src/components/InventoryV2/InventoryTable.tsx:1]: copied from V1

File: src/components/InventoryV2/Search.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
```
Comment (charlie-tf) [inline src/components/InventoryV2/Search.tsx:1]: copied from V1

**Review by constanzauanini — APPROVED** [review]
Looking good!

**Pull request merged**